cli/go/src/pxf-cli/cmd/cluster.go

```diff
@@ -54,6 +56,7 @@ func init() {
 	clusterCmd.AddCommand(startCmd)
 	clusterCmd.AddCommand(stopCmd)
 	clusterCmd.AddCommand(statusCmd)
+	syncCmd.Flags().BoolVarP(&NonDestructiveSync, "non-destructive", "n", false, "Don't delete files on remote host")
```

Comment:
maybe be more explicit here:
```suggestion
	syncCmd.Flags().BoolVarP(&NonDestructiveSync, "non-destructive", "n", false, "Do not delete files on segment hosts that have been deleted on master host")
```